tests/Cargo.toml

```diff
@@ -18,15 +18,18 @@ cpi = ["no-entrypoint"]
 default = []
 
 [dependencies]
-anchor-lang = "0.30.0"
+anchor-lang = "0.30.1"
 bincode = "1.3.3"
+borsh = "0.10.0"
 bv = "0.11.1"
 bytemuck = { version = "1.13.1", features = ["derive", "min_const_generics"] }
 cfg-if = "1.0.0"
 ed25519-dalek = "1.0.1"
 futures = "0.3.25"
 jito-steward = { features = ["no-entrypoint"], path = "../programs/steward" }
-jito-tip-distribution = { features = ["no-entrypoint"], git = "https://github.com/jito-foundation/jito-programs", rev = "50d450e993cb2278bcf97cd01b19e8a4f1f56e8e" }
+jito-tip-distribution = { features = ["no-entrypoint"], git = "https://github.com/jito-foundation/jito-programs", branch = "tj/priority-fee-distributor" }
```

Comment:
we'll want to pin these to a proper tag or commit when [this jito-programs](https://github.com/jito-foundation/jito-programs/pull/136) is punched into jito-programs